src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figures {
+    private int leftLeg;
+    private int rightLeg;
+
+    public RightTriangle(String color, int leftLeg, int rightLeg) {
+        super(color);
+        this.leftLeg = leftLeg;
+        this.rightLeg = rightLeg;
+    }
+
+    @Override
+    public double area() {
+        return (leftLeg * rightLeg) / 2;
+    }
+
+    @Override
+    public void draw() {

```

Comment:
The method name `draw` is correct, but it seems like it's not following the Single Responsibility Principle (SRP). This method is responsible for both drawing the figure and printing information about it. Consider splitting this functionality into two separate methods.